docs/guide/extending_online_editor.md

```diff
@@ -255,6 +255,99 @@ In the example above, the template files for the front end could be:
 
 Templates for Content View in the Back Office would be `app/Resources/views/themes/admin/field_type/ezrichtext/custom_style/highlighted_word.html.twig` and `app/Resources/views/themes/admin/field_type/ezrichtext/custom_style/highlighted_block.html.twig` respectively (assuming Admin SiteAccess uses the `admin` theme).
 
+## Custom data attributes and classes
+
+You can add custom data attributes and CSS classes to elements in the Online Editor.
+
+The available elements are:
+
+- `embedinline`
+- `embed`
+- `formatted`
+- `heading`
+- `embedimage`
+- `ul`
+- `ol`
+- `li`
+- `paragraph`
+- `table`
+- `tr`
+- `td`
+
+### Custom data attributes
+
+Custom data attributes are configured under the `fieldtypes.ezrichtext.attributes` key.
+The configuration is SiteAccess-aware.
+
+A custom data attribute can belong to one of four types: `choice`, `boolean`, `string`, or `number`.
+You can also set each attribute to be `required` and set its `default_value`.
+
+For the `choice` type, you must provide an array of available `choices`.
+Adding `multiple` enables you to choose whether more than one option can be selected.
```

Comment:
Note: `multiple` option proved to be confusing, so right now it's by default `false`, but only for data attributes. For classes it remains `true` by default.